src/window.rs

```diff
@@ -283,6 +286,8 @@ impl Window {
     ) -> WindowBuilder {
         use glutin::os::unix::WindowBuilderExt;
 
+        let icon= Icon::from_bytes_with_format(WINDOW_ICON, ImageFormat::ICO).unwrap();
```

Comment:
We can just pass this to the `get_platform_window` method, then we don't have to duplicate this code either.